qiita_ware/test/test_processing_pipeline.py

```diff
@@ -271,6 +326,55 @@ def test_get_preprocess_fastq_cmd(self):
         self.assertEqual(obs_cmd_1, exp_cmd_1)
         self.assertEqual(obs_cmd_2, exp_cmd_2)
 
+    def test_get_preprocess_fastq_cmd_per_sample_FASTQ(self):
+        metadata_dict = {
+            'SKB8.640193': {'run_prefix': "sample1", 'primer': 'A',
+                            'barcode': 'A', 'center_name': 'ANL',
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'A',
+                            'experiment_design_description': 'A'},
+            'SKD8.640184': {'run_prefix': "sample2", 'primer': 'A',
+                            'barcode': 'A', 'center_name': 'ANL',
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'A',
+                            'experiment_design_description': 'A'}}
+        md_template = pd.DataFrame.from_dict(metadata_dict, orient='index')
+        prep_template = PrepTemplate.create(md_template, Study(1), '16S')
+
+        tmp_dir = mkdtemp()
```

Comment:
What I mean is that the setup of a partial function that builds temporary paths (`self.path_builder`) is all being done in `setUp` ([here](https://github.com/antgonza/qiita/blob/issue-1230/qiita_ware/test/test_processing_pipeline.py#L74-75) and [here](https://github.com/antgonza/qiita/blob/issue-1230/qiita_ware/test/test_processing_pipeline.py#L107)), thus you don't need to run this again. From looking at the code, it doesn't seem like there are parts that are sensitive to the contents of the directory, so it would be safe to use as is.